README.md

```diff
@@ -4,18 +4,16 @@ Join our online chat at [![Gitter chat](https://badges.gitter.im/resin-io/chat.p
 
 This is [resin.io](https://resin.io)'s supervisor, a program that runs on IoT devices and has the task of running user Apps (which are Docker containers), and updating them as resin.io's API informs it to.
 
-The supervisor is for now a Node.js program, with a subset of its functionality implemented in Go.
-
-We are using [waffle.io](https://waffle.io) to manage our tickets / issues, so if you want to track our progress or contribute take a look at [our board there](https://waffle.io/resin-io/resin-supervisor).
+The supervisor is a Node.js program.
 
 ## Running a supervisor locally
 
 This process will allow you to run a development instance of the supervisor on your local computer. It is not recommended for production scenarios, but allows someone developing on the supervisor to test changes quickly.
-The supervisor is run inside a Docker-in-Docker container that roughly emulates its environment when running on a resinOS device.
+The supervisor is run inside a resinOS instance running in a container, so effectively it's a Docker-in-Docker instance (or more precisely, [balena](https://github.com/resin-os/balena)-in-Docker).
 
 ### Set up `config.json`
 
-Add a `tools/dind/config.json` file from a staging or production device image. It should be configured for an x86 or amd64 device type (e.g. an Intel Nuc) so that you can push apps to it and they run properly on your computer.
+Add a `tools/dind/config.json` file from a staging or production device image. It should be configured for an x86 or amd64 device type (e.g. an Intel NUC) so that you can push apps to it and they run properly on your computer.
```

Comment:
Can we clarify the steps here? Something like "Copy the `config.json` file from the `resin-boot` partition of a staging or production device image into the`tools/dind/`  directory. You can also download this file directly from the dashboard. Make sure you choose an x86 or amd64 device type, such as the Intel NUC, ..."

Reply:
There were steps below but I've rearranged this to hopefully make it clearer.